.dotnet/src/Custom/Assistants/FileSearchToolResources.cs

```diff
@@ -14,7 +14,7 @@ public partial class FileSearchToolResources
     public IList<string> VectorStoreIds
     {
         get => _vectorStoreIds;
-        init
+        set
```

Comment:
This one is also a collection.

Reply:
Let's fix it in a separate PR - https://github.com/joseharriaga/openai-in-typespec/pull/182#discussion_r1693640061